index.html

```diff
@@ -0,0 +1,219 @@
+                    <li class="bar1"></li>
+                    <li class="bar1"></li>
+                </ul>
+                <ul class="menue_bar_desktop">
+                    <li class="nav_list"><a href="#" class="nav_link">Portofolio</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">About</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="main__wrapper">
+            <div class="porto_wrapper">
+                <h1 class="headline">I'm Addisalem Glad to see you!</h1>
+                <p class="description">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+                <h2 class="connect_wrapper"><a class="contact" href="#">LETS CONNECT</a></h2>
+                <ul class="contact_us_wrapper1">
+                    <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+                    <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    
+    <section class="grid__wrapper">
+        <div class="covers">
+            <img class="img__class" id="pic__2" src="./img/Snap.png" alt="">
+            <img class="img__class" id="pic__1" src="./img/pic1.png" alt="">
+            <div class="right__box">
+                <h1 class="bold-text">Tonic</h1>
+                <ul class="selections">
+                    <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>CANOPY</li>
+                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                    <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
+                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                    <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+                </ul>
+                <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+                <ul class="platform">
+                    <li class="pla__btn">html</li>
+                    <li class="pla__btn">css</li>
+                    <li class="pla__btn">javascript</li>
+                </ul>
+                <button class="see-project">See Project</button>
```

Comment:
Please use a bit of small padding for the `See Project` button. KIndly see the screenshot below for reference.
![btn](https://user-images.githubusercontent.com/35504149/134526924-1e470c48-417a-40ae-81d8-857d9fe0da22.png)